### PR TITLE
Add Chromium backend build instructions

### LINKED ISSUE #1437: Instructions for building the Chromium AARs and Chromium version of Wolvic?
**Is your feature request related to a problem? Please describe.**
I'm interested in building the Chromium backed version of Wolvic from source. Could you please provide some instructions/guidance?

**Describe the solution you'd like**
Short README explaining how to build the required AARs and Wolvic with the chromium backend.

### PR DESCRIPTION
**Pull request by svillar**
As the README.md file is already way too crowded we've decided to add them in a separate file at least until we don't reorganize the documentation.

Fixes #1437

### CODE REVIEW

**Comment by felipeerias** [comment]
I tried the instructions to compile Chromium from scratch and they were very incomplete. Perhaps we are better off telling people to follow the official guide to compile Chromium and then add our fork.

**Comment by svillar** [comment]
> I tried the instructions to compile Chromium from scratch and they were very incomplete. Perhaps we are better off telling people to follow the official guide to compile Chromium and then add our fork.

Agree I totally forgot to add a link to the chromium official build instructions

**Comment by felipeerias** [comment]
The following sequence worked for me with a fresh install of Chromium.

Reference: https://chromium.googlesource.com/chromium/src/+/main/docs/linux/build_instructions.md

* Install depot tools and add them to the path
* `mkdir chromium && cd chromium`
* `fetch --nohooks --no-history chromium`
* edit `./.gclient`

```
solutions = [
  {
    "name": "src",
    "url": "https://chromium.googlesource.com/chromium/src.git",
    "managed": False,
    "custom_deps": {},
    "custom_vars": {
      "checkout_pgo_profiles": True, 
    },
  },
]
target_os = ["android"]
```

* `cd src`
* `./build/install-build-deps.sh`
* `gclient runhooks`
* `gn gen out/Default`
* edit `./out/Default/args.gn` (debug or release version)

```
target_os = "android"
target_cpu = "arm64"
is_official_build = true
ffmpeg_branding="Chrome"
proprietary_codecs=true
```

* `git remote add wolvic-chromium https://github.com/Igalia/wolvic-chromium`
* `git fetch wolvic-chromium`
* `git fetch --depth=1 wolvic-chromium`  to skip getting the history
  - alternatively, use something like `git fetch --depth=1288 wolvic-chromium` to get the commits that are new in our branch
* `git switch wolvic`
* `autoninja -C out/Default content_aar ui_aar`
  - here I had to add the PGO setting above and run `gclient sync`, and then compile again
* if that last command fails
  - run `autoninja -C out/Default content/shell:content_shell_resources_grit`
  - then run `autoninja -C out/Default content_aar ui_aar` again

After the compilation has finished successfully, continue with the steps to integrate the libraries into Wolvic.

File: CHROMIUM.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+_Set `local.properties`:_
+
+```
+chromium_aar = ${WHERE_PREBUILT_AARS_ARE}
+```
+
+_Choose build variant:_
+
+After adding `chromium_aar` in user.properties and syncing gradle, you can see various build variants with Chromium suffix. Please choose one among them.
```
Comment (felipeerias) [inline CHROMIUM.md:109]: It should be `local.properties`.

**Review by felipeerias — CHANGES_REQUESTED** [review]

**Comment by felipeerias** [comment]
@svillar I have added a commit with more details about the process of downloading and building Chromium from scratch.

**Review by felipeerias — APPROVED** [review]
LGTM, waiting for @svillar's feedback regarding the new additions.

**Pull request merged**